### DISCUSSION TITLE
how to hide timeline border and scroller

### DISCUSSION THREAD

**Discussion by deepghuge36** [andrewsuzuki/react-twitter-widgets#54] [Q&A] [answered]
like this

<TwitterTimelineEmbed
  sourceType="timeline"
  id="539487832448843776"
  theme="dark"
  noHeader
  noFooter
  options={{height: 400}}
/>

noHeader | bool | - | - | Hide the header from timeline
-- | -- | -- | -- | --
noFooter | bool | - | - | Hide the footer from timeline
noBorders | bool | - | - | Hide the border from timeline
noScrollbar | bool | - | - | Hide the scrollbars

**Comment by andrewsuzuki** [comment]
See the readme. Put options like `noBorders: true` in the options object.

**Comment by deepghuge36** [comment]
> See the readme. Put options like `noBorders: true` in the options object.

i'm missing something ?
`import "./styles.css";
import { Timeline } from "react-twitter-widgets";
export default function App() {
  return (
    <div className="App">
      <Timeline
        dataSource={{
          sourceType: "profile",
          screenName: "TwitterDev"
        }}
        options={{
          height: "400",
          noBorders: true,
          noFooter: true,
          noHeader: true,
          noScrollbar: true
        }}
      />
    </div>
  );
}
`
https://codesandbox.io/s/interesting-solomon-deqyx?file=/src/App.js

**Comment by andrewsuzuki** [comment, marked as answer]
I was using your example parameters but they don't seem to exist. See the Twitter documentation. Use `chrome: "noborders"` with space-separated values

**Comment by andrewsuzuki** [comment]
See the Twitter documentation, all the information is there. https://developer.twitter.com/en/docs/twitter-for-websites/timelines/guides/parameter-reference

**Comment by deepghuge36** [comment]
> See the Twitter documentation, all the information is there. https://developer.twitter.com/en/docs/twitter-for-websites/timelines/guides/parameter-reference

  chrome: 'noheader,nofooter,noscrollbar,noborders',
  
  thank you it's Done
